libGitWrap/Private/DiffListPrivate.hpp

```diff
@@ -36,6 +36,7 @@ namespace Git
         {
         public:
             explicit DiffListPrivate(const RepositoryPrivate::Ptr& repo, git_diff* diff);
+            explicit DiffListPrivate(RepositoryPrivate* repo, git_diff* diff);
```

Comment:
Neither one of these constructors ought to be marked `explicit`. Why is the second form required?